.github/workflows/publish-on-release.yml

```
@@ -64,3 +64,13 @@ jobs:
         run: upload-assets --url-path vanilla-framework-version-$(cat css/VANILLA_VERSION).min.css css/build.css
         env:
           UPLOAD_ASSETS_API_TOKEN: ${{secrets.UPLOAD_ASSETS_API_TOKEN}}
+
+  publish-website:
+    name: Publish to the production website
+    needs: [publish-assets]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Push to release branch  
+        run: |
+          git push origin release
```

@samhotep  Maybe it would make sense to force push? To make sure there are no conflicts or anything.
```suggestion
          git push --force origin release
```

---

Good idea! Added.